services/hanke-service/src/main/resources/db/changelog/changesets/086-change-car-lane-nuisance-enum.sql

```diff
@@ -0,0 +1,134 @@
+--liquibase formatted sql
+--changeset Teemu Hiltunen:086-change-car-lane-nuisance-enum
+--comment: Change 'hankealue.kaistahaitta' enum values and 'tormaystarkastelutulos.kaistapituushaitta' and 'applications.applicationData.areas[n].tyoalueet[m].tormaystarkastelutulos.kaistapituushaitta' index values
+
+-- update hankealue.kaistahaitta enum values
+ALTER TABLE hankealue
+    ADD COLUMN kaistahaitta_temp VARCHAR;
+
+UPDATE hankealue
+    SET kaistahaitta_temp =
+        CASE
+            WHEN kaistahaitta = 'VAHENTAA_KAISTAN_YHDELLA_AJOSUUNNALLA' THEN 'YKSI_KAISTA_VAHENEE'
+            WHEN kaistahaitta = 'VAHENTAA_SAMANAIKAISESTI_KAISTAN_KAHDELLA_AJOSUUNNALLA' THEN 'YKSI_KAISTA_VAHENEE_KAHDELLA_AJOSUUNNALLA'
+            WHEN kaistahaitta = 'VAHENTAA_SAMANAIKAISESTI_USEITA_KAISTOJA_KAHDELLA_AJOSUUNNALLA' THEN 'USEITA_KAISTOJA_VAHENEE_AJOSUUNNILLA'
+            WHEN kaistahaitta = 'VAHENTAA_SAMANAIKAISESTI_USEITA_KAISTOJA_LIITTYMIEN_ERI_SUUNNILLA' THEN 'USEITA_AJOSUUNTIA_POISTUU_KAYTOSTA'
+        END;
+
+ALTER TABLE hankealue
+    DROP COLUMN kaistahaitta;
+
+ALTER TABLE hankealue
+    RENAME COLUMN kaistahaitta_temp TO kaistahaitta;
+
+-- update tormaystarkastelutulos.kaistapituushaitta
+UPDATE tormaystarkastelutulos
```

Comment:
I don't think we should update the törmäystarkastelutulos. We're not recalculating the indexes, so they appear miscalculated after this. The recalculation will fix these values when someone edits the hanke.

Reply:
But there might be non-existing indices for kaistapituushaitta=1. Does it not matter if such values are shown until tormaystarkastelutulos is recalculated?

Reply:
Do we map the result back to the enum somewhere? That's the one thing that would be an issue.

Reply:
I don' think we do. There is no function for that.

Reply:
So we don't need to update the törmäystarkastelutulos.

Reply:
Removed changes to tormaystarkastelutulos